packages/discovery-react-components/src/components/DocumentPreview/components/PdfViewerHighlight/PdfViewerWithHighlight.stories.scss

```diff
@@ -0,0 +1,25 @@
+.withTextSelection {
+  display: flex;
+  height: 800px;
+
+  .rightPane {
+    flex: 1 1 auto;
+    width: 20%;
+    overflow-y: scroll;
+
+    p {
+      margin-bottom: 0.5rem;
+    }
+  }
+  .text {
+    overflow-wrap: break-word;
+    white-space: pre-wrap;
+    font-size: 10pt;
+    font-family: 'Courier New', Courier, monospace;
+  }
+
+  .highlight {
+    opacity: 0.4;
+    background: rgba(255, 64, 128, 1);
```

Comment:
We should check what Carbon says about highlighting text and which colors/opacity to use.

Reply:
Only thing I found so far is `$highlight` from https://www.carbondesignsystem.com/guidelines/color/usage. That's fine if there's a single highlight in the component, but we'll need more guidance if we have to handle multiple highlight colors.

Reply:
I update the code to use the `$highlight` color so far.